plugins/kubernetes/package.json

```diff
@@ -32,16 +32,19 @@
   },
   "dependencies": {
     "@backstage/catalog-model": "^0.7.3",
-    "@backstage/plugin-catalog-react": "^0.1.1",
     "@backstage/config": "^0.1.3",
     "@backstage/core": "^0.7.0",
+    "@backstage/core-api": "^0.2.12",
```

Comment:
Don't think you'll need to depend on this right? You should be able to grab it from `core` itself. I don't think anything should depend on `core-api` :)

Reply:
Will replace

Reply:
done